# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1250054447 -7200
#      Wed Aug 12 07:20:47 2009 +0200
# Node ID a85dbf7b6ff94b8fcc99c749caa13d99f1b63d64
# Parent  fb933db0c5171d4ffbb4a71b9fe20a9de340221e
fix some functions help markup

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2009-08-12  Jaroslav Hajek  <highegg@gmail.com>
+
+	* data.cc (Fissorted, F__sort_rows_idx__, Fnorm): Mark as Built-in
+	Functions in the inline help.
+
 2009-08-11  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/fftw.cc (Ffftw): Update for octave_fftw_planner
 	and octave_float_fftw_planner as singleton objects.
 
 2009-08-11  John W. Eaton  <jwe@octave.org>
 
 	* oct-conf.h.in: Use READLINE_LIBS instead of LIBREADLINE.
diff --git a/src/data.cc b/src/data.cc
--- a/src/data.cc
+++ b/src/data.cc
@@ -4788,17 +4788,17 @@ DEFUN (full, args, ,
 
   return retval;
 }
 
 // Compute various norms of the vector X.
 
 DEFUN (norm, args, ,
   "-*- texinfo -*-\n\
-@deftypefn {Function File} {} norm (@var{a}, @var{p}, @var{opt})\n\
+@deftypefn {Built-in Function} {} norm (@var{a}, @var{p}, @var{opt})\n\
 Compute the p-norm of the matrix @var{a}.  If the second argument is\n\
 missing, @code{p = 2} is assumed.\n\
 \n\
 If @var{a} is a matrix (or sparse matrix):\n\
 \n\
 @table @asis\n\
 @item @var{p} = @code{1}\n\
 1-norm, the largest column sum of the absolute values of @var{a}.\n\
@@ -5820,17 +5820,17 @@ ordered lists.\n\
 // Sort the rows of the matrix @var{a} according to the order
 // specified by @var{mode}, which can either be `ascend' or `descend'
 // and return the index vector corresponding to the sort order.
 //
 // This function does not yet support sparse matrices.
 
 DEFUN (__sort_rows_idx__, args, ,
   "-*- texinfo -*-\n\
-@deftypefn {Function File} {} __sort_rows_idx__ (@var{a}, @var{mode})\n\
+@deftypefn {Built-in Function} {} __sort_rows_idx__ (@var{a}, @var{mode})\n\
 Undocumented internal function.\n\
 @end deftypefn\n")
 {
   octave_value retval;
 
   int nargin = args.length ();
   sortmode smode = ASCENDING;
 
@@ -5867,17 +5867,17 @@ Undocumented internal function.\n\
   else
     error ("__sort_rows_idx__: needs a 2-dimensional object");
 
   return retval;
 }
 
 DEFUN (issorted, args, ,
   "-*- texinfo -*-\n\
-@deftypefn {Function File} {} issorted (@var{a}, @var{rows})\n\
+@deftypefn {Built-in Function} {} issorted (@var{a}, @var{rows})\n\
 Returns true if the array is sorted, ascending or descending.\n\
 NaNs are treated as by @code{sort}.  If @var{rows} is supplied and\n\
 has the value \"rows\", checks whether the array is sorted by rows\n\
 as if output by @code{sortrows} (with no options).\n\
 \n\
 This function does not yet support sparse matrices.\n\
 @seealso{sortrows, sort}\n\
 @end deftypefn\n")
